Rule34/dist/metadata.json

```diff
@@ -6,7 +6,8 @@
   "service": "Rule34",
   "description": {
     "en": "Rule 34 - If it exists, there's porn of it. If not, start uploading!",
-    "de": "Regel 34 - Wenn es existiert, gibt es Porn davon. Falls nicht, fang an etwas hochzuladen!"
+    "de": "Regel 34 - Wenn es existiert, gibt es Porn davon. Falls nicht, fang an etwas hochzuladen!",
+    "zh_CN": "Regel 34 - 这里在搞黄色，如果没有，那就上传!"
```

Comment:
This should probs be Rule or the chinese translation of Rule not the german version